compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -803,6 +803,11 @@ object Trees {
     type ThisTree[-T >: Untyped] = Import[T]
   }
 
+  case class Export[-T >: Untyped] private[ast] (expr: Tree[T], selectors: List[untpd.ImportSelector])(implicit @constructorOnly src: SourceFile)
```

Comment:
Since they have the same fields and are handled in similar ways, Import and Export could have a common superclass, just like Apply and TypeApply inherit from GenericApply

Reply:
Ah, it looks like Export is not a DenotingTree unlike Import, but maybe it should? That would involve typing it with an ImportType just like Imports.

Reply:
Thats a good idea I'll try that thanks, currently they are typed as Any lol